debian/debian/postinst

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -ex
 
-mkdir /var/trash
+mkdir -p /var/trash
```

Comment:
Why?

Reply:
Because it already exists when `postinst` is called for `truenas` package.
```
+ mkdir /var/trash
mkdir: cannot create directory '/var/trash': File exists
dpkg: error processing package truenas (--configure):
 installed truenas package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 truenas
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Reply:
Thank you, I forgot about that feature of this flag :)